lib/commands.js

```diff
@@ -86,6 +86,7 @@ module.exports = function(config) {
       operations.loadConfig,
       operations.promptParameters,
       operations.confirmCreate,
+      operations.validateParametersHook,
```

Comment:
I think this should hook before the `.confirmCreate` step, right?

Reply:
yep, I was in two minds about this! I decided to go with after `.confirmCreate` rather than before to let folks confirm that the parameters look okay to them and in the `create` command, this is explicit after `.confirmCreate`. Does that sound convincing to you?

Reply:
The prompt in `.confirmCreate` is "Ready to create the stack?", not a prompt to confirm your parameter inputs. https://github.com/mapbox/cfn-config/blob/master/lib/commands.js#L573

I think this flow would be confusing:

1. enter parameter values
2. confirm that you're ready to create a new stack
3. get your parameter values rejected

Reply:
Right, I see what you mean. Presently, there is an assumption that if you're entering a value for a parameter you're confirming it. I can move this. 👍